### ISSUE TITLE
Spinner: wrong role (textbox instead of spinbutton)

### ISSUE THREAD

**Issue by csalmhof** [primefaces/primefaces#8396]
**Describe the defect**
Spinner should get the role `spinbutton` (as changed in https://github.com/primefaces/primefaces/issues/4822) but is set to `textbox`.

**Reproducer**
Problem is appearing everytime spinner is used. Also in the [Showcase for Spinner](https://www.primefaces.org/showcase/ui/input/spinner.xhtml) the role of the Spinner is `textbox`.

**Environment:**
 - PF Version: _11.0.0_
 - PF Theme: _ALL_
 - Affected browsers: _ALL_

**To Reproduce**
1. Go to https://www.primefaces.org/showcase/ui/input/spinner.xhtml
2. Open the DevTools
3. Search for the input-element of spinner
4. See that `role="textbox"`

**Expected behavior**
The role should be `spinbutton`.

**Actual behavior**
The role is `textbox`.

**Example XHTML**
```html
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
      xmlns:f="http://xmlns.jcp.org/jsf/core"
      xmlns:p="http://primefaces.org/ui"
      xmlns:h="http://xmlns.jcp.org/jsf/html">

    <h:head>
        <title>PrimeFaces Test</title>
        <h:outputScript name="test.js" />
    </h:head>
    <h:body>

        <p:spinner/>

    </h:body>
</html>
```

**Example Bean**
Not needed

**Comment by melloware** [comment]
Weird because there is this line of code in spinner.js
https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/resources/META-INF/resources/primefaces/spinner/spinner.js#L372
